src/ui/backend/canvas/animate.js

```diff
@@ -433,6 +433,7 @@ var Animator = exports.Animator = Class(Emitter, function () {
     }
     queue.length = 0;
     this._elapsed = 0;
+    this._isPaused = false;
```

Comment:
Hmm... I wonder if we should update the native core as well.
https://github.com/gameclosure/native-core/blob/9ed8d903d92ffa6e52af32e18540bbee8defed08/timestep/timestep_animate.cpp#L138

That appears to be the equivalent function and it also isn't clearing the paused flag.  But I know very little about that code base.